trompace/queries/templates.py

```diff
@@ -20,16 +20,30 @@
 language
 }}'''
 
+QUERY_ALL_TEMPLATE = '''{queryname}{{
+identifier
+name
+publisher
+contributor
+creator
+source
+description
+language
+}}'''
+
 
 
 def format_query(queryname: str, args: Dict[str, Any]):
     """Create a query to send to the Contributor Environment.
     Arguments:
         mqueryname: the name of the query to generate
-        args: a dictionary of field: value pairs to add to the mutation
+        args: a dictionary of field: value pairs to add to the query.
     Returns:
         A formatted query
     """
-
-    formatted_query = QUERY_TEMPLATE.format(queryname=queryname, parameters=make_parameters(**args))
+    if args:
+    	formatted_query = QUERY_TEMPLATE.format(queryname=queryname, parameters=make_parameters(**args))
```

Comment:
I gave a [suggestion here](https://github.com/trompamusic/trompa-ce-client/pull/13#discussion_r377595803) about how we could use a single query template for both queries with parameters and default "all" queries. I'd prefer that we use this.

Reply:
Yes, I had implemented that. But the problem is that it leaves an empty bracket in the string which is not accepted the the CE.

Reply:
maybe my previous comment wasn't clear - 
we don't have to have the () inside the template. Currently we have `template = "{queryname}({params}){{ return values}}"`, but we can change it to `template = "{queryname}{params}{{ return values}}"`.
this way you can make the `parmams` substitution `"({})".format(make_parameters(whatever))` when there are parameters, or `""` when there are no args.